src/commands/login.ts

```diff
@@ -34,13 +34,13 @@ export const login = async (
   args: { org: string },
   options?: { skipAuthenticate?: boolean }
 ) => {
-  const orgDoc = await getDoc<RawOrgData, object>(
-    publicDoc(`orgs/${args.org}`)
-  );
+  await saveConfig(args.org);
+  await initializeFirebase();
```

Comment:
I don't follow why we need to add `initializeFirebase` here/. Wasn't it already initialized if we were able to read the doc before?

Reply:
There are two Firebase, the bootstrap one (which is configured to point to p0-prod) and the regular one, which uses whatever is specified in the config.